.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+python:
+  - "3.6.2"
+
+services:
+  - postgresql
+
+install:
+  - pip install -r requirements.txt
+
+script:
+  - coverage run --source=authors/ manage.py test
+
+after_success:
+  - coveralls
+
+deploy:
+  provider: heroku
+  api_key:
+    secure: ebc475a8-e613-4b9c-8adc-58b408c546c3
```

Comment:
you are exposing this to everyone. Please add this to the environmental variables on travis such that it can be read from there